kolibri/utils/sanity_checks.py

```diff
@@ -42,3 +44,14 @@ def check_port_availability(host, port):
         )
         s.close()
         sys.exit(1)
+
+
+def check_content_directory_exists():
+    content_directory = OPTIONS['Paths']['CONTENT_DIR']
+
+    # Check if the content directory exists
+    if not os.path.exists(content_directory):
+        logger.error(
+            "The content directory {} does not exist.".format(content_directory)
+        )
+        sys.exit(1)
```

Comment:
Should this also check if it's writable?

Reply:
yes I will implement that into the function!